abi/forum.abi

```diff
@@ -139,35 +210,35 @@
       ]
     },
     {
-      "name": "vote",
+      "name": "voterow",
       "base": "",
       "fields": [
         {
-          "name": "voter",
-          "type": "account_name"
-        },
-        {
-          "name": "proposer",
-          "type": "account_name"
+          "name": "id",
+          "type": "uint64"
         },
         {
           "name": "proposal_name",
           "type": "name"
         },
         {
-          "name": "proposal_hash",
-          "type": "string"
+          "name": "voter",
+          "type": "account_name"
         },
         {
           "name": "vote",
-          "type": "bool"
+          "type": "uint8"
```

Comment:
Assuming this is to allow more complex voting combinations.

Default/Standard 1=true & 0=false

Reply:
Exact, for now we would stick to what you described, that was proposed Daniel, so we decided to implement it, remains BC.